roles/ceph-common/tasks/main.yml

```diff
@@ -45,6 +45,54 @@
 - set_fact:
     is_ceph_infernalis={{ (ceph_stable and ceph_stable_release not in ceph_stable_releases) or (ceph_stable_rh_storage and (rh_storage_version.stdout | version_compare('0.94', '>'))) }}
 
+- set_fact:
+    dir_mode: 0755
+    dir_owner: ceph
+    dir_group: ceph
+  when: is_ceph_infernalis
+
+- set_fact:
+    dir_mode: 0755
+    dir_owner: root
+    dir_group: root
+  when: not is_ceph_infernalis
+
+- set_fact:
+    key_owner: root
+    key_group: root
+    key_mode: 0600
+  when: not is_ceph_infernalis
+
+- set_fact:
+    key_owner: ceph
+    key_group: ceph
+    key_mode: 0600
+  when: is_ceph_infernalis
+
+- set_fact:
+    activate_file_owner: ceph
+    activate_file_group: ceph
+    activate_file_mode: 0644
+  when: is_ceph_infernalis
+
+- set_fact:
+    activate_file_owner: root
+    activate_file_group: root
+    activate_file_mode: 0644
+  when: not is_ceph_infernalis
+
+- set_fact:
+    rbd_client_dir_mode: 0644
```

Comment:
for consistency purpose, can you move the mode after the group?
 Add a line note